Marlin/pins.h

```diff
@@ -432,4 +432,19 @@
     X_MS1_PIN, X_MS2_PIN, Y_MS1_PIN, Y_MS2_PIN, Z_MS1_PIN, Z_MS2_PIN \
   }
 
+#define HAS_DIGIPOTSS (PIN_EXISTS(DIGIPOTSS))
```

Comment:
With current codebase, you'll need to add `#include "fastio.h"` to the top of `pins.h`.